examples/manifests/deployment-with-tls.yaml

```diff
@@ -74,16 +75,28 @@ spec:
           name: tenants
           readOnly: true
           subPath: tenants.yaml
-        - mountPath: /mnt/tls
-          name: tls
+        - mountPath: /mnt/tls/cert
```

Comment:
Can we mount to `/var/run/secrets/tls` instead of `/mnt/tls`?

Reply:
How about we do `/var/run/tls`? that was we can leave the CA in the same directory even though it's not a secret

Reply:
why CA is from configmap?

Reply:
because it's not a secret and is potentially shared with many other Pods that should not be mounting the TLS certs